handlers/secrets_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/openfaas/faas/gateway/requests"
+	types "github.com/openfaas/faas-provider/types"
```

Comment:
Nit: what's the reason for this change?

Reply:
A compilation error :-) this package moved, see the notes I posted in #faas-provider on Slack